lib/store/index.js

```diff
@@ -70,6 +72,27 @@ class Store {
 
     this.grammar = grammar;
   }
+
+  @action setBusySignalRegistry(registry) {
+    this.busySignalRegistry = registry;
+    this.runningKernels.forEach(this._registerBusySignal);
+  }
+
+  _registerBusySignal(kernel: Kernel) {
+    if (this.busySignalRegistry) {
+      let busySignalProvider = this.busySignalRegistry.create();
```

Comment:
@lgeiger I'd expect eslint to catch that this variable declaration should be a `const`. Would it be OK to update .eslintrc to report this?